.scalafmt.conf

```diff
@@ -1,23 +1,45 @@
-version = 2.7.5
+version = 3.0.8
 
 style = default
 
 maxColumn = 100
+docstrings.wrap = false
```

Comment:
Is this for minimization of the current diff?

I haven't configured my editor to default to the new style, so I'm not that fond of it, but I'm fond of defaults.  I could go either way on this.

Reply:
Mostly that, yeah. Enabling wrapping also broke some doctests, but I wasn't on the latest version when I tried it so I could try again.

Reply:
Ah yes, a couple of doctests are failing to compile now.

Reply:
Okay, let's leave it false then.  I would love to see the number of doctests grow.